services/svc-pinger/test/ip.scheme.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'vitest'
+import { ZodError } from 'zod'
+import { ipv4Scheme } from '../src/schemes/ip.scheme'
+
+const invalidAddresses = ["123", "002.001", "999.999.999.999", "qwerty.com", "266.266.266.266"]
+const validAddresses = ["172.16.254.1:8000", "192.168.0.1", "11.22.33.44", "172.16.254.1:8000/code?as=123"]
+
+describe("IPv4 scheme", () => {
```

Comment:
To avoid unnecessary duplication of tests, the schema should be tested within the integration tests of corresponding endpoints